resources/styles/mixins/index.less

```diff
@@ -60,6 +60,14 @@
   z-index: 2;
 }
 
+.sans(@size: 1.5rem, @line-height: 3rem) {
+  font-family: 'Lato', Helvetica, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: @size;
+  line-height: @line-height;
+}
+
```

Comment:
I had to do this the on some PR or another as well since we don't have the `#font` mixins.  Do you think we should move them over as well?  Probably don't need to right away, but something we should think about.